torch/onnx/symbolic_opset11.py

```diff
@@ -530,6 +530,12 @@ def _get_arange_dtype(dtype):
         raise NotImplementedError("Unknown aten::arange signature taking " + str(len(args)) + " arguments.")
     return arange_tensor
 
+def linspace(g, start, end, steps, dtype, layout, device, pin_memory):
+    dtype = sym_help._maybe_get_const(dtype, "i")
+    step = div(g, sub(g, end, start), sub(g, steps, g.op("Constant", value_t=torch.tensor(1, dtype=torch.int64))))
+    end_epsilon = g.op("Add", step, end)
+    type, end, start, step = sym_help._arange_cast_helper(g, start=start, end=end_epsilon, step=step, dtype=dtype)
+    return g.op("Range", start, end, step)
```

Comment:
There is some duplication of code with opset 9 linspace. Seems a `arange` helper may come in handy such that only need to define linspace in opset 9?

Reply:
I think we should be fine just using the opset 9 symbolic. Nice catch :)

Reply:
Updated by adding helper function